src/getting_started/nargo/commands.md

```diff
@@ -47,21 +47,12 @@ _Arguments_
 
 Generate a Solidity verifier smart contract for the program.
 
-## `nargo compile [FLAGS] <circuit_name>`
+## `nargo compile <circuit_name>`
 
-Compile the program and its secret execution trace into [ACIR](../../acir.md) format.
+Compile the program and its secret execution trace into a JSON build artifact file containing the ACIR representation, and the ABI of the circuit. This build artifact can then be used to generate and verify proofs.
 
 _Arguments_
 
-- `<circuit_name>` - The name of the ACIR file
-- `[FLAGS]` - Add `--witness` to compile the witness file as well
+- `<circuit_name>` - The name of the circuit file
 
-_Usage_
-
-Running the command would create a new folder `build` with the compiled `<circuit_name>.acir` file in your project directory.
-
-To also compile a witness file, fill in the values in `Prover.toml` generated from `nargo check` and run the command with the `--witness` flag. A `<circuit_name>.tr` file would be compiled in the `build` folder.
```

Comment:
Why was the flags section removed? It seems the `--witness` flag is outdated, but we also have the `--circuits` flag now which could replace it

Reply:
Sorry Jake, this was merged already before your review… Shall we raise a new issue, or revert this PR?

Reply:
We can just make a new issue/pr

Reply:
The changes related to `witness` are made in https://github.com/noir-lang/book/pull/57